src/components/navBar/index.tsx

```diff
@@ -1,17 +1,131 @@
-import Image from 'next/image';
-import styles from '@/components/navBar/navBar.module.scss';
-
-const RDSLogo = '/RDSLogo.png';
-
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+import Image from "next/image";
+import styles from "@/components/navBar/navBar.module.scss";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { USER_SELF } from "../constants/url";
+import navLinks from "./navLinks";
+
+const RDSLogo = "/RDSLogo.png";
+const GitHubLogo = "/gitHubLogo.png";
+const hamburgerLogo = "/hamburger.svg";
+const DEFAULT_AVATAR = "/dummyProfile.png";
+const SIGN_IN_WITH_GITHUB = "Sign In With GitHub";
+const SIGN_IN = "Sign In";
+
+const NavBar = () => {
+  const [toggle, setToggle] = useState(false);
+  const [userData, setUserData] = useState({});
+  const [userisLoggedIn, setUserisLoggedIn] = useState(false);
+
+  useEffect(() => {
+    (async function () {
+      try {
+        const { data, status, statusText, incompleteUserDetails } =
+          await axios.get(USER_SELF, { withCredentials: true });
```

Comment:
Is it possible to use the existing [fetch](https://github.com/Real-Dev-Squad/website-status/blob/develop/src/helperFunctions/fetch.ts) helper function?
Reference - https://github.com/Real-Dev-Squad/website-status/blob/develop/src/pages/index.tsx#L57

Reply:
maybe but do we need a helper function just to make that call ?

Reply:
Done ✅